### PR TITLE
Add ability to run tasks from a managed task definition

### PR DESCRIPTION
**Pull request by fishnix**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -105,7 +108,82 @@ GET /v1/ecs/{account}/clusters/{cluster}/tasks/{task}
 POST /v1/ecs/{account}/taskdefs
 GET /v1/ecs/{account}/clusters/{cluster}/taskdefs
 DELETE /v1/ecs/{account}/clusters/{cluster}/taskdefs/{taskdef}
-GET /v1/ecs/{account}/clusters/{cluster}/taskdefs/{taskdef}
+GET /v1/ecs/{account}/clusters/{cluster}/taskdefs/{taskdef}- [ECS-API](#ecs-api)
```
Comment (btassone) [inline README.md:111]: @fishnix - I think a rogue paste got by ya 😄

**Review by btassone — APPROVED** [review]
Gave it a whirl. Everything seems to work, all tests pass.

**Comment by damnski** [comment]
:+1:

**Review by FishNecks — APPROVED** [review]
👍

**Review by FishNecks — APPROVED** [review]
👍

**Pull request merged**